cli-feed-v3.json

```diff
@@ -104,6 +104,16 @@
       "sha2": "A148DC9C2783D2E7E5E50F59BE2EAEA7948213A30876F41171CF20E9F84754F2",
       "FUNCTIONS_EXTENSION_VERSION": "beta"
     },
+    "2.2.0": {
+      "Microsoft.NET.Sdk.Functions": "1.0.13",
+      "cli": "https://functionscdn.azureedge.net/public/2.0.1-beta.30/Azure.Functions.Cli.no-runtime.2.0.1-beta.30.zip",
+      "localEntryPoint": "func.exe",

```

Comment:
the `no-runtime` package doesn't have a `func.exe`. @soninaren  is VS using the `localEntryPoint` now? 

Btw, the self-contained builds are not signed. I expect VS to not like that.

Reply:
VS is using the localEntry Point with the latest update. https://github.com/Azure/app-service-announcements/issues/118. Let me do a quick test to see if it causes issues with unsigned bits.

Reply:
thanks @soninaren, @alrod update the `cli` to use the `win-x86` package instead of the `no-runtime` one.

Reply:
I tested it, looks like unsigned bits are working. We should be good to go.